lib/solidus_braintree/engine.rb

```diff
@@ -4,6 +4,10 @@
 require 'solidus_support'
 
 module SolidusBraintree
+  def self.table_name_prefix
```

Comment:
any reason we don't put this in the model(s) class(es). maybe add a baserecord class and let rails do its thing?

Reply:
We can, that's definitely an option! A module that has this that's included in the models would do the same, so happy to move to that if we prefer.

I am not sure why we didn't do it that way, but overriding the class method is suggested in some Rails docs. I discovered by [some posts on SO](https://stackoverflow.com/questions/8911046/rails-table-name-prefix-missing?source=post_page-----a08199deaa69--------------------------------) other people have had issues with this and auto-loading, but mostly on old Rails (pre 4.x).

Reply:
To dig into this a little deeper... maybe we should just be really explicit and define each `table_name` per model instead of dealing with `table_name_prefix`. For example:

```diff
 module SolidusBraintree
   class Source
+    self.table_name = "solidus_paypal_braintree_sources"
```

This leaves even less room for application developers to be confused about why the table names don't match the models.

Reply:
I like that.

Reply:
@tvdeyen We've updated the PR here to reflect that, let us know if this looks like a better approach to you.